src/Services/DataValueServices.php

```diff
@@ -48,6 +49,26 @@
  */
 return [
 
+	/**
+	 * UnitConverter
+	 *
+	 * @return callable
+	 */
+	'UnitConverter' => function( $containerBuilder ) {
```

Comment:
I think the MW services style approach for object graph construction, which is used here, is not so good maintenance wise. No time to describe this now, shoot me a mail if you want some more explanation